Update R template - Migrated to a self hosted repo by the community author (#1673)
microsoft/vscode-dev-containers: containers/r/README.md

> **IMPORTANT NOTE: The template `R` is migrated to [rocker-org/devcontainer-templates](https://github.com/rocker-org/devcontainer-templates/tree/main/src/r-ver) by the community author.**

Reviewer: ```suggestion
**IMPORTANT NOTE: The `R` template is now hosted by the community author in the repo [rocker-org/devcontainer-templates](https://github.com/rocker-org/devcontainer-templates/tree/main/src/r-ver).**
```
